components/director/internal/domain/apptemplate/converter.go

```diff
@@ -95,12 +95,18 @@ func (c *converter) InputFromGraphQL(in graphql.ApplicationTemplateInput) (model
 		return model.ApplicationTemplateInput{}, errors.Wrapf(err, "error occurred while converting webhooks og GraphQL input to Application Template model with name %s", in.Name)
 	}
 
+	var labels map[string]interface{}
+	if in.Labels != nil {
+		labels = in.Labels
+	}
+
 	return model.ApplicationTemplateInput{
 		Name:                 in.Name,
 		Description:          in.Description,
 		ApplicationInputJSON: appCreateInput,
 		Placeholders:         c.placeholdersFromGraphql(in.Placeholders),
 		AccessLevel:          model.ApplicationTemplateAccessLevel(in.AccessLevel),
+		Labels:               labels,
```

Comment:
What about UpdateInputFromGraphQL, if labels are updated they will not be applied without adoption there?
The same is for From/To entity functions too.

Reply:
after offline discussion, we've decided to implement updating of labels in additional task